content/src/roadmaps/add-ons/nonprofit.json

```diff
@@ -12,6 +12,13 @@
       "weight": 3,
       "task": "nonprofit-business-trustees"
     },
+    {
+      "step": 2,
+      "weight": 1,
+      "task": "certificate-good-standing-foreign",
+
+      "required": true
+    },
     {
```

Comment:
Boulder: So adding this to the nonprofit.json roadmap will make that show up for all nonprofit legal structures not just Dakotas.